backend/src/main/java/botobo/core/quiz/domain/card/Cards.java

```diff
@@ -17,6 +19,19 @@
     @OneToMany(mappedBy = "workbook", cascade = CascadeType.PERSIST, orphanRemoval = true)
     private List<Card> cards = new ArrayList<>();
 
+    public Cards(List<Card> cards) {
+        this.cards = cards;
+    }
+
+    public Cards shuffle() {
+        Collections.shuffle(cards);
+        return this;
+    }
+
+    public List<Card> subList(int size) {
+        return cards.stream().limit(size).collect(Collectors.toList());
+    }
+
```

Comment:
```suggestion
public List<Card> getCardsForQuiz() {
        Collections.shuffle(cards);
        List<Card> quizzes = cards.stream()
                .filter(Card::isNextQuiz)
                .limit(10)
                .collect(Collectors.toList());
        quizzes.forEach(Card::cancelNextQuiz);
        if (quizzes.size() < 10) {
            List<Card> remainder = cards.stream()
                    .filter(card -> !card.isNextQuiz())
                    .limit(10 - quizzes.size())
                    .collect(Collectors.toList());
            quizzes.addAll(remainder);
        }
        return quizzes;
    }
```
대충 이런 코드를 넣어서 isNextQuiz가 true인 카드 먼저 뽑고 10개가 아닐 때 그 외 나머지 수 만큼 false인 카드를 뽑고 합쳐서 반환하게 하고싶은데 스트림 실력이 부족해서 많이 아쉽네요ㅠㅠ
더 좋은 코드가 분명 있을텐데....

Reply:
좋네요 👍
isNextQuiz == true 카드가 10장 이하일 때 quizzes.forEach(Card::cancelNextQuiz); 부분이 먼저 실행되어서
isNextQuiz == false인 카드에서 중복으로 뽑힐 수 있겠네요